src/EventSubscriber/StrawberryfieldEventInsertSubscriberDepositDO.php

```diff
@@ -91,6 +98,9 @@ public function __construct(
     $this->destinationScheme = $config_factory->get(
       'strawberryfield.storage_settings'
     )->get('object_file_scheme');
+    $this->destinationRelativePath = $config_factory->get(
+      'strawberryfield.storage_settings'
+    )->get('object_file_path') ?? "dostorage";
```

Comment:
This looks good. What if you make `dostorage` a class constant?

Reply:
Done.